src/amo/components/AddonReviewListItem/index.js

```diff
@@ -54,6 +56,20 @@ type InternalProps = {|
 |};
 
 export class AddonReviewListItemBase extends React.Component<InternalProps> {
+  onClickToDeleteReview = (event: SyntheticEvent<HTMLElement>) => {
+    const { dispatch, errorHandler, isReplyToReviewId, review } = this.props;
+    event.preventDefault();
+
+    invariant(review, 'review is required');
```

Comment:
just curious, does invariant need to be here and in actions/reducer? If its ok, should we add errorHandler too? it seems that's required too, right?

Reply:
Because we've told Flow that `review` can be undefined or null (it's declared as `review?: UserReviewType | null`), we need to use `invariant` here to tell Flow that we never expect it to be undefined or null at this point, and if it is an exception should be thrown.

Reply:
ok I think I see..thanks : )